File: hrms/src/main/java/com/nberimen/employer/EmployerService.java
package com.nberimen.employer;

import java.util.List;

import com.nberimen.utilities.result.DataResult;
import com.nberimen.utilities.result.Result;

public interface EmployerService {

	public DataResult<List<Employer>> getAll();
	public Result register(Employer employer);
}

File: hrms-frontend/src/services/employerService.js
import axios from 'axios'

export default function EmployerService(){
    getAllEmployers = () => {
        return axios.get("http://localhost:8080/api/employers/getall");
    }
    registerEmployer = employer => {
        return axios.post("http://localhost:8080/api/employers/register", employer);
    }
}